### PR TITLE
[BUG] - Fix broken and missing links

### LINKED ISSUE #206: [BUG] - 404 Not found pages
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

These pages are returning a 404 not found status code: 

https://www.nebari.dev/docs/docs/troubleshooting#handle-access-to-restricted-namespaces
https://www.nebari.dev/docs/how-tos/docs/how-tos/domain-registry
https://www.nebari.dev/docs/how-tos/docs/troubleshooting
https://www.nebari.dev/how-tos/nebari-gcp#deploying-nebari
https://www.nebari.dev/docs/how-tos/docs/get-started/installing-nebari
https://www.nebari.dev/docs/how-tos/docs/tutorials
https://www.nebari.dev/docs/how-tos/docs/how-tos/nebari-azure#nebari-initialize
https://www.nebari.dev/docs/how-tos/docs/how-tos/nebari-do#nebari-initialize
https://www.nebari.dev/docs/get-started/docs/how-tos/nebari-local
https://www.nebari.dev/docs/get-started/docs/get-started/cloud-providers
https://www.nebari.dev/docs/how-tos/docs/how-tos/nebari-aws#nebari-initialize
https://www.nebari.dev/docs/how-tos/docs/how-tos/nebari-gcp#nebari-initialize

It seems something is adding a redundant `/docs/` or `/docs/how-tos/` folders somehow.

These pages are linked to from these pages: 

|    | url                                                                      | not found links                                                                                   |
|---:|:-------------------------------------------------------------------------|:----------------------------------------------------------------------------------------|
|  0 | https://www.nebari.dev/docs/troubleshooting/                             | https://www.nebari.dev/docs/docs/troubleshooting#handle-access-to-restricted-namespaces |
|  1 | https://www.nebari.dev/docs/how-tos/domain-registry/                     | https://www.nebari.dev/how-tos/nebari-gcp#deploying-nebari                              |
|  2 | https://www.nebari.dev/docs/how-tos/domain-registry/                     | https://www.nebari.dev/how-tos/nebari-gcp#deploying-nebari                              |
|  3 | https://www.nebari.dev/docs/how-tos/nebari-azure/                        | https://www.nebari.dev/docs/how-tos/docs/get-started/installing-nebari                  |
|  4 | https://www.nebari.dev/docs/how-tos/nebari-azure/                        | https://www.nebari.dev/docs/how-tos/docs/troubleshooting                                |
|  5 | https://www.nebari.dev/docs/how-tos/nebari-azure/                        | https://www.nebari.dev/docs/how-tos/docs/tutorials                                      |
|  6 | https://www.nebari.dev/docs/how-tos/nebari-azure/                        | https://www.nebari.dev/docs/how-tos/docs/how-tos/nebari-azure#nebari-initialize         |
|  7 | https://www.nebari.dev/docs/how-tos/nebari-azure/                        | https://www.nebari.dev/docs/how-tos/docs/how-tos/domain-registry                        |
|  8 | https://www.nebari.dev/docs/how-tos/nebari-do/                           | https://www.nebari.dev/docs/how-tos/docs/get-started/installing-nebari                  |
|  9 | https://www.nebari.dev/docs/how-tos/nebari-do/                           | https://www.nebari.dev/docs/how-tos/docs/troubleshooting                                |
| 10 | https://www.nebari.dev/docs/how-tos/nebari-do/                           | https://www.nebari.dev/docs/how-tos/docs/tutorials                                      |
| 11 | https://www.nebari.dev/docs/how-tos/nebari-do/                           | https://www.nebari.dev/docs/how-tos/docs/how-tos/nebari-do#nebari-initialize            |
| 12 | https://www.nebari.dev/docs/how-tos/nebari-do/                           | https://www.nebari.dev/docs/how-tos/docs/how-tos/domain-registry                        |
| 13 | https://www.nebari.dev/docs/get-started/deploy/                          | https://www.nebari.dev/docs/get-started/docs/get-started/cloud-providers                |
| 14 | https://www.nebari.dev/docs/get-started/deploy/                          | https://www.nebari.dev/docs/get-started/docs/how-tos/nebari-local                       |
| 15 | https://www.nebari.dev/docs/how-tos/nebari-aws/                          | https://www.nebari.dev/docs/how-tos/docs/get-started/installing-nebari                  |
| 16 | https://www.nebari.dev/docs/how-tos/nebari-aws/                          | https://www.nebari.dev/docs/how-tos/docs/troubleshooting                                |
| 17 | https://www.nebari.dev/docs/how-tos/nebari-aws/                          | https://www.nebari.dev/docs/how-tos/docs/tutorials                                      |
| 18 | https://www.nebari.dev/docs/how-tos/nebari-aws/                          | https://www.nebari.dev/docs/how-tos/docs/how-tos/nebari-aws#nebari-initialize           |
| 19 | https://www.nebari.dev/docs/how-tos/nebari-aws/                          | https://www.nebari.dev/docs/how-tos/docs/how-tos/domain-registry                        |
| 20 | https://www.nebari.dev/docs/how-tos/nebari-gcp/                          | https://www.nebari.dev/docs/how-tos/docs/get-started/installing-nebari                  |
| 21 | https://www.nebari.dev/docs/how-tos/nebari-gcp/                          | https://www.nebari.dev/docs/how-tos/docs/troubleshooting                                |
| 22 | https://www.nebari.dev/docs/how-tos/nebari-gcp/                          | https://www.nebari.dev/docs/how-tos/docs/tutorials                                      |
| 23 | https://www.nebari.dev/docs/how-tos/nebari-gcp/                          | https://www.nebari.dev/docs/how-tos/docs/how-tos/nebari-gcp#nebari-initialize           |
| 24 | https://www.nebari.dev/docs/how-tos/nebari-gcp/                          | https://www.nebari.dev/docs/how-tos/docs/how-tos/domain-registry                        |
| 25 | https://www.nebari.dev/docs/how-tos/docs/how-tos/domain-registry         | https://www.nebari.dev/docs/how-tos/docs/how-tos/domain-registry                        |
| 26 | https://www.nebari.dev/docs/how-tos/docs/how-tos/domain-registry         | https://www.nebari.dev/docs/how-tos/docs/how-tos/domain-registry                        |
| 27 | https://www.nebari.dev/docs/how-tos/docs/troubleshooting                 | https://www.nebari.dev/docs/how-tos/docs/troubleshooting                                |
| 28 | https://www.nebari.dev/docs/how-tos/docs/troubleshooting                 | https://www.nebari.dev/docs/how-tos/docs/troubleshooting                                |
| 29 | https://www.nebari.dev/docs/how-tos/docs/get-started/installing-nebari   | https://www.nebari.dev/docs/how-tos/docs/get-started/installing-nebari                  |
| 30 | https://www.nebari.dev/docs/how-tos/docs/get-started/installing-nebari   | https://www.nebari.dev/docs/how-tos/docs/get-started/installing-nebari                  |
| 31 | https://www.nebari.dev/docs/how-tos/docs/tutorials                       | https://www.nebari.dev/docs/how-tos/docs/tutorials                                      |
| 32 | https://www.nebari.dev/docs/how-tos/docs/tutorials                       | https://www.nebari.dev/docs/how-tos/docs/tutorials                                      |
| 33 | https://www.nebari.dev/docs/get-started/docs/how-tos/nebari-local        | https://www.nebari.dev/docs/get-started/docs/how-tos/nebari-local                       |
| 34 | https://www.nebari.dev/docs/get-started/docs/how-tos/nebari-local        | https://www.nebari.dev/docs/get-started/docs/how-tos/nebari-local                       |
| 35 | https://www.nebari.dev/docs/get-started/docs/get-started/cloud-providers | https://www.nebari.dev/docs/get-started/docs/get-started/cloud-providers                |
| 36 | https://www.nebari.dev/docs/get-started/docs/get-started/cloud-providers | https://www.nebari.dev/docs/get-started/docs/get-started/cloud-providers                |

### Steps to Resolve this Issue

1. Check how the wrong URL path parts are being added (script, manually, etc.)
2. Remove it

### LINKED ISSUE #211: [BUG] - Get Started dropdown in header missing content
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

The `Get Started` dropdown in the header is missing content/doesn't match when comparing it to the sidebar. 

image

### Steps to Resolve this Issue

1.
2.
3.
...

### PR DESCRIPTION
**Pull request by trallard**
- :pencil2: Fix typos in community/
- :bug: Fix links in get-started/
- :bug: Fix how-tos links
- :pencil2: Improvements to tutorials/
- :pencil2: Improvements to docs/
- :pencil2: Fix more links in how-tos/
- :pencil2: Use internal links
- :wrench: Add Home link to header
- :pencil2: Update internal links

## Reference Issues or PRs

Closes #211 
Closes #206 

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: docs/docs/how-tos/kbatch-howto.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Wrote config to /home/<username>/.config/kbatch/config.json
 When submitting a job to `kbatch`, there are a handful of required arguments you will need to provide.
 These arguments can be passed directly from the command-line, or you can create a configuration file and pass that to `kbatch`.
 
-For more details on how to use `kbatch` please refer to [the kbatch documentation page](https://kbatch.readthedocs.io/en/latest/user-guide.html).
+For more details on how to use `kbatch` please refer to [the `kbatch`s documentation page](https://kbatch.readthedocs.io/en/latest/user-guide.html).
```
Comment (pavithraes) [inline docs/docs/how-tos/kbatch-howto.md:68]:
```suggestion
For more details on how to use `kbatch` please refer to [the `kbatch` documentation page](https://kbatch.readthedocs.io/en/latest/user-guide.html).
```

File: docs/docs/how-tos/nebari-aws.md [hunk 2]
```diff
@@ -91,14 +91,14 @@ startup file (for example, for example in the `~/.bashrc` or `~/.profile` for th
 :::
 
 :::note
-The steps in the following sections assume you have (i) completed the [Install Nebari](docs/get-started/installing-nebari) section, (ii) confirmed that Nebari is successfully
+The steps in the following sections assume you have (i) completed the [Install Nebari][nebari-install] section, (ii) confirmed that Nebari is successfully
 installed in your environment, (iii) opted for **AWS** as your cloud provider, and (iv) already configured the Nebari environment variables. If you had any issues during the
-installation, please visit the "Get started" section of our [troubleshooting page](docs/troubleshooting) for further guidance.
+installation, please visit the "Get started" section of our [troubleshooting page][nebari-troubleshoothing] for further guidance.
```
Comment (pavithraes) [inline docs/docs/how-tos/nebari-aws.md:96]:
```suggestion
installation, please visit the "Get started" section of our [troubleshooting page][nebari-troubleshooting] for further guidance.
```

File: docs/docs/how-tos/nebari-aws.md [hunk 3]
```diff
@@ -228,3 +232,9 @@ Nebari also has a `destroy` command that works the same way the deploy works but
 ```bash
 nebari destroy -c nebari-config.yaml
 ```
+
+<!-- internal links -->
+
+[nebari-install]: /get-started/installing-nebari.md
+[nebari-troubleshoothing]: /troubleshooting.mdx
```
Comment (pavithraes) [inline docs/docs/how-tos/nebari-aws.md:239]:
```suggestion
[nebari-troubleshooting]: /troubleshooting.mdx
```

File: docs/docs/how-tos/nebari-azure.md [hunk 4]
```diff
@@ -74,14 +74,14 @@ startup file (for example, for example in the `~/.bashrc` or `~/.profile` for th
 :::
 
 :::note
-The steps in the following sections assume you have (i) completed the [Install Nebari](docs/get-started/installing-nebari) section, (ii) confirmed that Nebari is successfully
+The steps in the following sections assume you have (i) completed the [Install Nebari][nebari-install] section, (ii) confirmed that Nebari is successfully
 installed in your environment, (iii) opted for **Azure** as your cloud provider, and (iv) already configured the Nebari environment variables. If you had any issues during the
-installation, please visit the "Get started" section of our [troubleshooting page](docs/troubleshooting) for further guidance.
+installation, please visit the "Get started" section of our [troubleshooting page][nebari-troubleshoothing] for further guidance.
```
Comment (pavithraes) [inline docs/docs/how-tos/nebari-azure.md:79]:
```suggestion
installation, please visit the "Get started" section of our [troubleshooting page][nebari-troubleshooting] for further guidance.
```

File: docs/docs/how-tos/nebari-azure.md [hunk 5]
```diff
@@ -211,3 +214,9 @@ Nebari also has a `destroy` command that works the same way the deploy works but
 ```bash
 nebari destroy -c nebari-config.yaml
 ```
+
+<!-- internal links -->
+
+[nebari-install]: /get-started/installing-nebari.md
+[nebari-troubleshoothing]: /troubleshooting.mdx
```
Comment (pavithraes) [inline docs/docs/how-tos/nebari-azure.md:221]:
```suggestion
[nebari-troubleshooting]: /troubleshooting.mdx
```

File: docs/docs/how-tos/nebari-do.md [hunk 6]
```diff
@@ -70,14 +72,14 @@ These environment variables will apply only to your current shell session. If yo
 startup file (for example, for example in the `~/.bashrc` or `~/.profile` for the bash shell). You can also opt for [`direnv`](https://direnv.net/) as a shell extension for managing your environment variables.
 :::
 
-:::note The steps in the following sections assume you have (i) completed the [Install Nebari](docs/get-started/installing-nebari) section, (ii) confirmed that Nebari is successfully
+:::note The steps in the following sections assume you have (i) completed the [Install Nebari][nebari-install] section, (ii) confirmed that Nebari is successfully
 installed in your environment, (iii) opted for **Digital Ocean** as your cloud provider, and (iv) already configured the Nebari environment variables. If you had any issues
-during the installation, please visit the "Get started" section of our [troubleshooting page](docs/troubleshooting) for further guidance.
+during the installation, please visit the "Get started" section of our [troubleshooting page][nebari-troubleshoothing] for further guidance.
```
Comment (pavithraes) [inline docs/docs/how-tos/nebari-do.md:77]:
```suggestion
during the installation, please visit the "Get started" section of our [troubleshooting page][nebari-troubleshooting] for further guidance.
```

File: docs/docs/how-tos/nebari-do.md [hunk 7]
```diff
@@ -207,3 +210,9 @@ Nebari also has a `destroy` command that works the same way the deploy works but
 ```bash
 nebari destroy -c nebari-config.yaml
 ```
+
+<!-- internal links -->
+
+[nebari-install]: /get-started/installing-nebari.md
+[nebari-troubleshoothing]: /troubleshooting.mdx
```
Comment (pavithraes) [inline docs/docs/how-tos/nebari-do.md:217]:
```suggestion
[nebari-troubleshooting]: /troubleshooting.mdx
```

File: docs/docs/get-started/quickstart.md [hunk 8]
```diff
@@ -252,4 +252,8 @@ This command is automatically run when you `deploy`.
 
 ---
 
-If you face any issues with the commands, check out the [Troubleshooting guide](../troubleshooting.mdx).
+If you face any issues with the commands, check out the [Troubleshooting guide][nebari-troubleshoothing].
```
Comment (pavithraes) [inline docs/docs/get-started/quickstart.md:255]:
```suggestion
If you face any issues with the commands, check out the [Troubleshooting guide][nebari-troubleshooting].
```

File: docs/docs/get-started/quickstart.md [hunk 9]
```diff
@@ -252,4 +252,8 @@ This command is automatically run when you `deploy`.
...
+
+<!-- internal links -->
+[install-nebari]: /get-started/installing-nebari.md
+[nebari-troubleshoothing]: /troubleshooting.mdx
```
Comment (pavithraes) [inline docs/docs/get-started/quickstart.md:259]:
```suggestion
[nebari-troubleshooting]: /troubleshooting.mdx
```

File: docs/docs/how-tos/nebari-gcp.md [hunk 10]
```diff
@@ -111,15 +111,15 @@ startup file (for example, for example in the `~/.bashrc` or `~/.profile` for th
 :::
 
 :::note
-The steps in the following sections assume you have (i) completed the [Install Nebari](docs/get-started/installing-nebari) section, (ii) confirmed that Nebari is successfully
+The steps in the following sections assume you have (i) completed the [Install Nebari][nebari-install] section, (ii) confirmed that Nebari is successfully
 installed in your environment, (iii) opted for **GCP** as your cloud provider which includes installing and initializing `gcloud`, and (iv) already configured the Nebari
-environment variables. If you had any issues during the installation, please visit the "Get started" section of our [troubleshooting page](docs/troubleshooting) for further
+environment variables. If you had any issues during the installation, please visit the "Get started" section of our [troubleshooting page][nebari-troubleshoothing] for further
```
Comment (pavithraes) [inline docs/docs/how-tos/nebari-gcp.md:116]:
```suggestion
environment variables. If you had any issues during the installation, please visit the "Get started" section of our [troubleshooting page][nebari-troubleshooting] for further
```

File: docs/docs/how-tos/nebari-gcp.md [hunk 11]
```diff
@@ -246,8 +247,14 @@ nebari destroy --help
...
+Nebari also has a `destroy` command that works the same way the deployment works but instead of creating the provisioned resources it destroys it.
 
 ```bash
 nebari destroy -c nebari-config.yaml
 ```
+
+<!-- internal links -->
+
+[nebari-install]: /get-started/installing-nebari.md
+[nebari-troubleshoothing]: /troubleshooting.mdx
```
Comment (pavithraes) [inline docs/docs/how-tos/nebari-gcp.md:259]:
```suggestion
[nebari-troubleshooting]: /troubleshooting.mdx
```

**Review by pavithraes — APPROVED** [review]
Thank you, @trallard! This looks good! My suggestions are only a few typo fixes, so I'll approve and we can merge after. :)

**Comment by trallard** [comment]
Sorted, if you are happy @pavithraes  can you merge?

**Pull request merged**